scripts/prepare_sector_network.py

```diff
@@ -3771,6 +3772,9 @@ def add_geothermal_district_heating_supply(n, egs_potential):
     cooling_demand = pd.read_csv(
         snakemake.input.cooling_demand, index_col=0, header=[0, 1], parse_dates=True
     ).fillna(0)
+    #breakpoint()
+    #cooling_demand /= 3e9 #testing convergence by downscaling cooling demand
+    #cooling_demand.loc[:,:] = 0

```

Comment:
Do you prefer to keep those comments in the code? Could be good to add a comment on them to clarify the reason why they have been included. Otherwise, the comments tend to stay in the code forever since nobody is able to remember the reason why are they there

Reply:
Perfect, thank you 😄